docs/docs/forms.mdx

```diff
@@ -421,8 +421,11 @@ class ValidateSlots(Action):
 ### The requested_slot slot
 
 The slot `requested_slot` is automatically added to the domain as an
-unfeaturized slot. If you want to make it featurized, you need to add it
-to your domain file as a categorical slot. You might want to do this if you
+slot of type [text](domain.mdx#text-slot). The value of the `requested_slot` will be
+ignored during conversations. If you want to change this behavior, you need to add
+the `requested_slot`  to your domain file as a categorical slot with
+`influence_conversation` set to `true`.
```

Comment:
Noticing this here but in general - are slot types e.g. `text` code or not? They're inconsistently formatted

Reply:
I think formatting them as code is reasonable. Sometimes it just makes more sense to directly link to them. What other formatting have you seen?

Reply:
I've just seen some "it is a text slot" vs " it is a `text` slot"
They can be formatted as code & linked to:

```suggestion
slot of type [`text`](domain.mdx#text-slot). The value of the `requested_slot` will be
ignored during conversations. If you want to change this behavior, you need to add
the `requested_slot`  to your domain file as a categorical slot with
`influence_conversation` set to `true`.
```

Reply:
ah nice! Didn't know that